ui/css/module/_footnotes.scss

```diff
@@ -5,14 +5,38 @@
   padding: 2px 4px;
   font-size: 0.6em;
   margin-left: .3em;
+  position: relative;
 
   &.active {
-    border-color: $color-primary-darker;
-    background: $color-primary-darker;
-    color: $color-white;
+    @extend .active-footnote-style;
```

Comment:
Does a mixin make sense here? The concern I have with `extend` is that we have to create a "real" css class, which might trip us up later.